tests/Integration/Generators/MailMakeCommandTest.php

```diff
@@ -97,12 +98,11 @@ public function testItCanGenerateMailWithViewWithNoInitialInput()
     public function testItCanGenerateMailWithMarkdownViewWithNoInitialInput()
     {
         $this->artisan('make:mail')
-
             ->expectsQuestion('What should the mailable be named?', 'FooMail')
             ->expectsQuestion('Would you like to create a view?', 'markdown')
             ->assertExitCode(0);
 
-        $this->assertFilenameExists('app/Mail/MyFooMail.php');
-        $this->assertFilenameExists('resources/views/mail/my-foo-mail.blade.php');
+        $this->assertFilenameExists('app/Mail/FooMail.php');
+        $this->assertFilenameExists('resources/views/mail/foo-mail.blade.php');
```

Comment:
This assertion is incorrect since published files from the previous test weren't cleared correctly.